src/cmd/nsfs.js

```diff
@@ -317,7 +317,7 @@ async function main(argv = minimist(process.argv.slice(2))) {
             }
         });
         if (await endpoint.is_http_allowed(nsfs_config_root)) {
-            dbg.log0('nsfs: listening on', util.inspect(`http://localhost:${http_port}`));
+            console.log('nsfs: listening on', util.inspect(`http://localhost:${http_port}`));
```

Comment:
I wonder, why are we using console and not dbg? 
by requireing the console we "hijak" the console, but console does not have levels like 0,1 ...

Reply:
@liranmauda ya totally agree with you, we need to have a discussion on this. In this PR I just reverted the previous code that I accidentally updated in my last PR.